gslab_scons/README.md

```diff
@@ -1,18 +1,20 @@
 ## Notes on release.py
 
-Make a release from an SCons directory by running the following
-from the command line within the directory of interest (e.g. `gslab-econ/template/paper_slides`):
+Make a release from an SCons directory by running the module `release`. 
+
+A user can either run the following from the command line within the directory of interest (e.g. `gslab-econ/template/paper_slides`) or write a Python script to call the module (see "Python wrapper" section below). If using the command line, run the following:
```

Comment:
Make the template path a link.

Reply:
I changed this to `/paper_slides`. I think that makes more sense, we don't want to be updating this if our template organization changes.